Minesweeper.py

```diff
@@ -99,6 +99,8 @@ def game_loop():
             if board[player_pos[1]][player_pos[0]] == 0:
                 board[player_pos[1]][player_pos[0]] = "F"
                 flags += 1
+            elif board[player_pos[1]][player_pos[0]] == "F":  # Error: Redundant condition, already checked in unflag
```

Comment:
There is a redundant condition check because "F" was already verified in the previous if statement.
```diff
-             elif board[player_pos[1]][player_pos[0]] == "F":  # Error: Redundant condition, already checked in unflag
+             else:  # Change to else as the previous condition already handled board status
```
---